usd/src/UsdUtils.hh

```diff
@@ -45,38 +47,62 @@ namespace sdf
   //
   namespace usd
   {
-    /// \brief Get an object's pose w.r.t. its parent.
+    /// \brief Set an object's pose w.r.t. its parent.
```

Comment:
is this really `Set`? It seems more like it's getting the pose, since `_pose` is an output of this function